### PR TITLE
update generate-cases skill

### PR DESCRIPTION
**Pull request by olexandr13**
image

### CODE REVIEW

File: skills/generate-test-cases/SKILL.md [hunk 1]
```diff
@@ -157,28 +157,42 @@ I've gathered information from the following sources:
...
 ```markdown
-❓ What type of tests do you want to generate?
+❓ How much coverage do you want?
 
-1. smoke
-2. regression
-3. acceptance
-4. ✏️ other
+1. 🚀 Smoke — ~<N> tests — critical-path only
+2. ⚖️ Balanced — ~<N> tests — happy path, key negative and edge cases
```
Comment (mykhailiukVitalii) [inline skills/generate-test-cases/SKILL.md]: move edge cases to "Exhaustive" option?

File: skills/generate-test-cases/SKILL.md [hunk 2]
```diff
@@ -235,6 +249,8 @@ Choose **default** in case user does not specify role.
 
 Create a **hierarchical, categorized, well-structured checklist** based on the gathered information and user choices.
 
+**Size the checklist to match the scope tier chosen in Step 2** (🚀 smoke → only critical paths; ⚖️ balanced → happy + key negative/edge; 🧨 exhaustive → full coverage). For ✏️ other, follow what the user described.
```
Comment (mykhailiukVitalii) [inline skills/generate-test-cases/SKILL.md]: same: move edge cases to "Exhaustive" option?

**Comment by olexandr13** [comment]
https://github.com/testomatio/skills/issues/67
